core/domain/user_jobs_one_off.py

```diff
@@ -311,28 +331,36 @@ class UserProfilePictureOneOffJob(jobs.BaseMapReduceOneOffJobManager):
 
     @classmethod
     def entity_classes_to_map_over(cls):
+        """Return a list of datastore class references to map over."""
         return [user_models.UserSettingsModel]
 
     @staticmethod
     def map(item):
+        """Implements the map function for this job."""
         if item.deleted or item.profile_picture_data_url is not None:
             return
 
         user_services.generate_initial_profile_picture(item.id)
 
     @staticmethod
     def reduce(key, stringified_values):
+        """Implements the reduce function for this job."""
         pass
 
 
 class UserLastExplorationActivityOneOffJob(jobs.BaseMapReduceOneOffJobManager):
-
+    """One-off job that adds fields to record last exploration created and last
+    edited time, last_created_an_exploration and last_edited_an_exploration
```

Comment:
I don't think you need the part after the comma. That's more of an implementation detail.

Also, time --> times.

Reply:
done.